### ISSUE TITLE
To provide a possibility to change timezone.

### ISSUE THREAD

**Issue by dubilyer** [reportportal/reportportal#706]
Ability to change a timezone during the installation process or further in reportportal UI.
...

After deploying the docker compose, Report portal always starts with UTC timezone.
When creating a weekly based "launch statistics chart" widget, it shows bar per day of the week (see attached picture). Because of time difference (I located in UTC+3), it moves a part of launches to the previous day bar.
![Capture1](https://user-images.githubusercontent.com/16655093/64669669-23e04400-d46b-11e9-8523-ab805d11deee.PNG)
![Capture2](https://user-images.githubusercontent.com/16655093/64669670-2478da80-d46b-11e9-9d96-9a0d4d75d165.PNG)

...

**Comment by DzmitryHumianiuk** [comment]
@dubilyer i guess, you can manage it by changing time in environment, where report portal deployed.
Server returns data with his TimeZone shift. Since use it from environment.

Will it help?
https://serverfault.com/questions/683605/docker-container-time-timezone-will-not-reflect-changes
https://bobcares.com/blog/change-time-in-docker-container/

**Comment by dubilyer** [comment]
Thanks for your answer. For both environment I deployed it started with UDC. 
I've already checked those links before opening request))

To change timezones in all containers I need to override images (create my own dockerfiles based on each image, and run some additional commands, e.g. install tzdata on alpine and copy a proper zoneinfo file to /etc/timezone). This solution is a problematic one - maintenance + not fast-forward upgradability.

I managed to change a timezone on several containers (based on alpine with tzdata installed. api for example) by setting env variable TZ in a docker-compose file. But it doesn't work with others.

It seems that changing date only in api containers solves the issue (need to be tested on next two nightly runs to be sure). But I'm afraid to unsync the system... Have you an idea what can be affected?
